packages/web/src/components/user-list/components/UserList.module.css

```diff
@@ -1,8 +1,18 @@
 .user {
-  padding: 14px 12px;
+  padding: 16px;
   display: flex;
   justify-content: space-between;
-  align-items: center;
+  align-items: flex-start;
+  cursor: pointer;
+}
+.user:hover {
+  background: var(--neutral-light-10);
+}
+.user:active {
+  background: var(--neutral-light-9);
+}
+.user.notLastUser {
```

Comment:
Why not `.user:not(:last-child)`? Or have this for all users, but have the last user `.user:last-child` set `border-bottom: 0`?